S/solar/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "solar"
+version = v"0.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/bbopt/solar.git", "d0c2932a3f25d4b0a71010b87b1f929c6fc1e020"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/solar/src
+if [[ "${target}" == *mingw* ]]; then
+    atomic_patch -p1 "${WORKSPACE}/srcdir/patches/windows.patch"
+fi
+make
+mkdir -p $bindir
+if [[ "${target}" == *mingw* ]]; then
+    cp ../bin/solar.exe $bindir/solar.exe
+else
+  cp ../bin/solar $bindir/solar
+end
+exit
```

Comment:
We have [environment variables](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables) for a reason :slightly_smiling_face: 
```suggestion
cp "../bin/solar${exeext}" "${bindir}/solar${exeext}"
```